Formula/infracost.rb

```diff
@@ -0,0 +1,78 @@
+class Infracost < Formula
+  desc "Cost estimates for Terraform"
+  homepage "https://docs.infracost.io/"
+  url "https://github.com/infracost/infracost/archive/v0.4.1.tar.gz"
+  sha256 "d466a917de411020a828c59b555af3c9de1e1a686888b875e6b82432caf3c5c0"
+  license "Apache-2.0"
+  head "https://github.com/infracost/infracost.git"
+
+  depends_on "go" => :build
+  depends_on "terraform" => :test
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/infracost"
+    pkgshare.install "examples"
+  end
+
+  test do
+    expected = <<~EOS
+      [{
+        "resource": "aws_ebs_volume.storage_option_1",
+        "breakdown": [{
+          "priceComponent": "GB",
+          "quantity": "15",
+          "unit": "GB/month",
+          "hourlyCost": "0.002568",
+          "monthlyCost": "1.875"
+        }, {
+          "priceComponent": "IOPS",
+          "quantity": "100",
+          "unit": "IOPS/month",
+          "hourlyCost": "0.008904",
+          "monthlyCost": "6.5"
+        }]
+      }, {
+        "resource": "aws_ebs_volume.storage_option_2",
+        "breakdown": [{
+          "priceComponent": "GB",
+          "quantity": "15",
+          "unit": "GB/month",
+          "hourlyCost": "0.001027",
+          "monthlyCost": "0.75"
+        }]
+      }, {
+        "resource": "aws_instance.web_app",
+        "breakdown": [{
+          "priceComponent": "instance hours (t3.micro)",
+          "quantity": "730",
+          "unit": "hour",
+          "hourlyCost": "0.0104",
+          "monthlyCost": "7.592"
+        }],
+        "subresources": [{
+          "resource": "aws_instance.web_app.root_block_device",
+          "breakdown": [{
+            "priceComponent": "GB",
+            "quantity": "15",
+            "unit": "GB/month",
+            "hourlyCost": "0.002055",
+            "monthlyCost": "1.5"
+          }]
+        }]
+      }, {
+        "resource": "aws_nat_gateway.nat",
+        "breakdown": [{
+          "priceComponent": "Hours",
+          "quantity": "730",
+          "unit": "hour",
+          "hourlyCost": "0.045",
+          "monthlyCost": "32.85"
+        }]
+      }]
+    EOS
+
+    cp_r "#{pkgshare}/examples/small_terraform", testpath
```

Comment:
`examples/terraform` is the updated version in v0.5.1 (since we now have unit+integration tests, we only need one terraform example)

Reply:
I have rewrite the small the builtin example so that I can have a minimal test case for testing the utility.